convention-develocity-gradle-plugin/plugins/gradle-5-or-newer/src/main/java/com/myorg/ConventionDevelocityGradlePlugin.java

```diff
@@ -69,15 +67,14 @@ private boolean containsPropertiesTask(StartParameter startParameter) {
                 || startParameter.getTaskNames().stream().anyMatch(it -> it.endsWith(":properties"));
     }
 
-    private void configureBuildScan(BuildScanExtension buildScan) {
+    private void configureBuildScan(BuildScanConfiguration buildScan) {
         // CHANGE ME: Apply your Build Scan configuration here
-        buildScan.publishAlways();
-        buildScan.setUploadInBackground(!isCi());
+        buildScan.getUploadInBackground().set(isCi());
```

Comment:
I really wanted to update `isCi()` to use the Provider API and return a `Provider<Boolean>` but it just isn't worth it until the build cache configuration is updated to use Provider API.